src/main/java/mate/jdbc/service/ManufacturerServiceImpl.java

```diff
@@ -18,7 +19,7 @@ public Manufacturer create(Manufacturer manufacturer) {
 
     @Override
     public Manufacturer get(Long id) {
-        return manufacturerDao.get(id).get();
+        return manufacturerDao.get(id).orElseThrow(NoSuchElementException::new);
```

Comment:
You should put message into this exception. Include value of 'id' into this message